utils/getSmartContract.tsx

```diff
@@ -1,39 +1,108 @@
-import { ethers } from 'ethers'
+import { useContext } from 'react'
```

Comment:
I'm not sure what is the purpose of this file. Is it to get all contracts related to a pair? ie, uniswap pair, uni router, uni factory, tokens, gs factory, gs pool, gs pos mgr? If so, is it updating all revelavent contracts on token update?

Reply:
This was supposed to be a test file where I tried connecting smart contracts and testing out their functions. I realized that this was confusing to you all so I'll be removing it soon. In turn, I'll propose something that can conveniently solve this